qiita_ware/ebi.py

```diff
@@ -143,33 +126,76 @@ def __init__(self, preprocessed_data_id, action):
             self.new_investigation_type = it
         else:
             # This should never happen
-            raise ValueError("Unrecognized investigation type: '%s'. This "
-                             "term is neither one of the official terms nor "
-                             "one of the user-defined terms in the ENA "
-                             "ontology")
-
-        self.ebi_dir = join(qiita_config.working_dir, 'ebi_submission_',
-                            str(preprocessed_data_id))
-        self.sequence_files = []
-        self.study_xml_fp = None
-        self.sample_xml_fp = None
-        self.experiment_xml_fp = None
-        self.run_xml_fp = None
-        self.submission_xml_fp = None
+            error_msgs.append("Unrecognized investigation type: '%s'. This "
+                              "term is neither one of the official terms nor "
+                              "one of the user-defined terms in the ENA "
+                              "ontology." % it)
+
+        self.ebi_dir = join(qiita_config.working_dir,
+                            'ebi_submission_%d' % preprocessed_data_id)
+        get_output_fp = partial(join, self.ebi_dir, 'xml_dir')
+        self.xml_dir = get_output_fp('')
+        self.study_xml_fp = get_output_fp('study.xml')
+        self.sample_xml_fp = get_output_fp('sample.xml')
+        self.experiment_xml_fp = get_output_fp('experiment.xml')
+        self.run_xml_fp = get_output_fp('run.xml')
+        self.submission_xml_fp = get_output_fp('submission.xml')
         self.pmids = s.pmids
 
-        # dicts that map investigation_type to library attributes
-        lib_strategies = {'metagenome': 'POOLCLONE',
-                          'mimarks-survey': 'AMPLICON'}
-        lib_selections = {'mimarks-survey': 'PCR'}
-        lib_sources = {}
-
-        self.library_strategy = lib_strategies.get(key, "OTHER")
-        self.library_source = lib_sources.get(key, "METAGENOMIC")
-        self.library_selection = lib_selections.get(key, "unspecified")
-
-        # This will hold the submission's samples, keyed by the sample name
+        # ignore data type for the restrictions as we assume that this was
```

Comment:
I don't have a strong feeling about this just thought it was easier to remove than add. If nobody opposes I will only use EBI restrictions.